articles/why-python-is-essential-for-data-analysis/index.md

```diff
@@ -0,0 +1,84 @@
+# WHY PYTHON IS ESSENTIAL FOR DATA ANALYSIS
+
+![hero image](/engineering-education/why-python-is-essential-for-data-analysis/datascience.jpg)
+
+### Introduction
+
+Data is an essential part of any enterprise/business.
+It is critical to collect, process, and analyze data flow in a timely and accurate manner.
+ 
+The data science industry is rapidly expanding. Data volume can be large, making information handling difficult and time-consuming.
+
+Python is a popular programming language in scientific computing because it’s many data-oriented feature packages can speed up and simplify data processing, saving time.
```

Comment:
```suggestion
Python is a popular programming language in scientific computing because its many data-oriented feature packages can speed up and simplify data processing, thus saving time.
```